openhtf/io/proto/test_runs.proto

```diff
@@ -71,10 +188,10 @@ message TestRun {
   // The name of the tester where this run was started
   required string tester_name = 2;
   required TestInfo test_info = 3;
-  required Status test_status = 4;
+  required Status test_status = 4 [default = CREATED];
 
-  // The openhtf cell number where this test took place
-  optional int32 cell_number = 15 [deprecated=true];
+  // The xtf cell number where this test took place
```

Comment:
prefer to keep this deprecated in the OpenHTF version, as it's not a thing we really support.  Arguably, it should be deprecated in the google3 version, i'm not sure anyone still uses it in any meaningful way.